python/src/deltachat/tracker.py

```diff
@@ -90,11 +90,11 @@ def wait_progress(self, data1=None):
             if data1 is None or evdata == data1:
                 break
 
-    def wait_finish(self):
+    def wait_finish(self, timeout=None):
         """ wait until configure is completed.
 
         Raise Exception if Configure failed
         """
-        if not self._configure_events.get():
+        if not self._configure_events.get(timeout=timeout):
```

Comment:
maybe capture the queue.Empty error and raise a `TimeoutError` instead, also document it in the method docstring

Reply:
Thanks, good point.